consensus/istanbul/core/roundstate.go

```diff
@@ -378,6 +387,31 @@ func (s *roundStateImpl) PreparedCertificate() istanbul.PreparedCertificate {
 	return s.preparedCertificate
 }
 
+func (s *roundStateImpl) SetProposalVerificationStatus(proposalHash common.Hash, verificationStatus error) error {
```

Comment:
You don't need to return `error`. That was done to allow persistence to fail, but we are not persisting this